.github/workflows/ci.yml

```diff
@@ -58,6 +58,10 @@ jobs:
         with:
           working-directory: src/github.com/containerd/containerd
           repo-access-token: ${{ secrets.GITHUB_TOKEN }}
+          # github.com/cyphar/filepath-securejoin: MPL-2.0 AND BSD-3-Clause
+          # Approval: https://github.com/cncf/foundation/issues/1154#issuecomment-3562385979
+          go-licenses-ignore: |
```

Comment:
So once the whole approval is done, a line will be added to one of the CSV's in https://github.com/cncf/foundation/tree/9c7d1eaed8af8e8ac21c8ac546898ea47aa2f110/license-exceptions and after that this exception can be removed? Or do we continue have to add this exception everywhere?

Reply:
Opened an issue
- https://github.com/containerd/project-checks/issues/24